src/steps/podman.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::Result;
+
+use crate::terminal::print_separator;
+use crate::{execution_context::ExecutionContext, utils::require};
+use json;
+use log::debug;
+use std::path::Path;
+use std::process::Command;
+
+/// Returns a Vector of all containers, as a triple (repository, container name, version)
+fn list_containers(podman: &Path) -> Result<Vec<(String, String, String)>> {
+    let output = Command::new(podman).args(&["images", "--format", "json"]).output()?;
+    let output_str = String::from_utf8(output.stdout)?;
+    let json_str = json::parse(&output_str)?;
+
+    let mut retval = vec![];
+    for lentry in json_str.members() {
+        // FIXME: Bad unwraps!
```

Comment:
Maybe parsing the JSON with serde will be better.

Reply:
Much the same API in fact, but as `serde` is already a dependency, I now changed it to use `serde_json` rather than `json` (Which still adds a new dependency, unfortunately). I'll have to replace the JSON parsing with something else in any case to make it compatible with docker. I'll try my luck on this later today.